YesterdayApi/Core/Identity/Validators/UserRegistrationRequestValidator.cs

```diff
@@ -0,0 +1,16 @@
+using FluentValidation;
+using YesterdayApi.Core.Identity.Web;
+using YesterdayApi.Utilities.AutomaticDI;
+
+namespace YesterdayApi.Core.Identity.Validators
+{
+    public class UserRegistrationRequestValidator : AbstractValidator<UserRegistrationRequest>, ITransient
+    {
+        public UserRegistrationRequestValidator()
+        {
+            Include(new UserIdentityRequestValidator());
+            RuleFor(urr => urr.Email).NotNull();
+            RuleFor(urr => urr.Email).Length(7, 230);
```

Comment:
Weird validator. Wouldn't be this better if you'll try this? What's about "x@a.pl (there's a domain like that 🤣)

```c#
RuleFor(urr => urr.Email).NotEmpty().MaxLength(230);
```